src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -0,0 +1,284 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.MarkdownLite
+{
+    using System.Collections.Immutable;
+
+    public class JsonTokenTreeRenderer
+    {
+        public ImmutableDictionary<string, string> Tokens { get; set; }
```

Comment:
forget to remove it , it was going to be used in dfm token

---
In reply to: [74891198](https://github.com/dotnet/docfx/pull/608#discussion_r74891198) [](ancestors = 74891198)